recipes/mykrobe/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: mykrobe
+  version: "0.5.6"
+
+source:
+  fn: mykrobe-0.5.6.tar.gz
+  url: https://pypi.python.org/packages/7e/35/a662ec560cfc91f35519e1459de917001ca19a9dbefd53c5b3a2b14122d0/mykrobe-0.5.6.tar.gz
+  md5: 5e912497f98cc192e3f0fbf426210560
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - mykatlas
+    - ga4ghmongo
+    - pyvcf
+    - mongoengine
+    - biopython
+    - future
+
+  run:
+    - python
+    - mykatlas
+    - ga4ghmongo
+    
+test:
+  # Python imports
+  imports:
+    - mykrobe
+  commands:
+    - mykrobe --help
+
+
+about:
+  home: https://github.com/iqbal-lab/Mykrobe-predictor
+  license: https://github.com/iqbal-lab/Mykrobe-predictor/blob/master/LICENSE
+  summary: '.'
```

Comment:
Even though there doesn't seem to be an "official" description in the projects's `setup.py` or `README.md`, can you add a short one?

Reply:
Fixed